.goreleaser.nightly.yml

```diff
@@ -133,7 +134,7 @@ kos:
       - "quay.io/authzed/spicedb-git"
       - "ghcr.io/authzed/spicedb-git"
       - "authzed/spicedb-git"
-    base_image: "docker.io/busybox"
+    base_image: "cgr.dev/chainguard/busybox"
```

Comment:
This is just updating it to be the same as the other `base_image` keys. I missed this when reviewing.